include/file.iol

```diff
@@ -68,23 +68,30 @@ type ListRequest:void {
 	.order?: void {
 		.byname?: bool
 	}
+	.fileInfo?: bool // it returns also file infos. Default is false
 }
 
 type ListResponse:void {
-	.result[0,*]:string
+	.result[0,*]:string {
```

Comment:
I skipped the fileInfo node just because it will be more easy on the client side to process the nodes. But I can insert it.